src-ui/src/app/components/document-list/bulk-editor/bulk-editor.component.ts

```diff
@@ -33,10 +35,12 @@ export class BulkEditorComponent {
   tags: PaperlessTag[]
   correspondents: PaperlessCorrespondent[]
   documentTypes: PaperlessDocumentType[]
+  storagePathes: PaperlessStoragePath[]
 
   tagSelectionModel = new FilterableDropdownSelectionModel()
   correspondentSelectionModel = new FilterableDropdownSelectionModel()
   documentTypeSelectionModel = new FilterableDropdownSelectionModel()
+  storagePathesSelectionModel = new FilterableDropdownSelectionModel()
   awaitingDownload: boolean
 
   constructor(
```

Comment:
Oh, by the way I have a concern about this. My understanding is that editing a storage path doesnt actually move it in the backend, is that correct? If so we need to figure out how to handle this because as-is a user would not know this. Presumably there needs to be a very explicit warning somewhere, or we just actually dont allow changing the path via the frontend but that seems like too much of a limitation.

Reply:
Agree. Same limitation as of today when you adjust the storage layout. It’s in the docs.

Probably add this as a sentence to the „what to put here“ hin.

Reply:
I added a note to the dialog as I think its not obvious otherwise. We can have a separate discussion about automatically running `document_renamer`, I agree its a separate question.

Reply:
👍. And even the task can be added easily by oneself via the admin interface. It doesn’t require a code change.